blaseball_mike/models.py

```diff
@@ -1081,13 +1081,14 @@ def order(self, value):
 
 
 class Idol(Base):
+    _player = None
```

Comment:
this is going to end up defining a class variable instead of an instance variable. You'll want to override `__init__` and instantiate there, or you can change line 1096 to be `getattr(self, '_player', None)`